docs/content/docs/data-structures/overview.md

```diff
@@ -40,12 +40,11 @@ Below we've enumerated the data structures and described when they may be most u
 
 ## Simple data object
 
-Use the [SharedCell]({{< relref "/docs/apis/cell/sharedcell.md" >}}) DDS when you simply need to wrap a data object,
-whether it is a number, string, or any other data type. You can also use a complex data type, such as an array or
-JavaScript object, in a SharedCell, but the value of a SharedCell can be changed only whole-for-whole; so this is not an
-appropriate choice if the application's clients will be changing subparts of the value, such as specific properties of
-an object or specific items in an array. In the latter scenarios, consider separating the data items with one of the
-key-value DDSes.
+Use the [SharedCell][] DDS when you simply need to wrap a data object, whether it is a number, string, or any other data
```

Comment:
No relref?

Reply:
It looks like the links still work though 😄  How is this still correctly linking the md files?

Reply:
uses the Links section below :O seems good for longer docs with lots of links like this.  not as helpful for the shorter ones we wrote 🤔

Reply:
Helio's correct. I'm on a quest for "smart links" so you can just write `[Term we use often[][]` to link to the canonical place for that term. The "links" section is the initial attempt at that.